hovercontrol/src/homie.rs

```diff
@@ -0,0 +1,209 @@
+use crate::config::{get_mqtt_options, MqttConfig};
+use crate::controller::{
+    DEFAULT_MAX_SPEED, DEFAULT_SCALE, DEFAULT_SPRING_CONSTANT, MAX_MAX_SPEED, MAX_SCALE,
+    MAX_SPRING_CONSTANT, SPRING_CONSTANT_STEP,
+};
+use eyre::Report;
+use homie_device::{HomieDevice, Node, Property};
+use log::{error, trace};
+use messages::{Side, SpeedLimits};
+use tokio::runtime::Handle;
+
+const HOMIE_PREFIX: &str = "homie";
+const HOMIE_DEVICE_ID: &str = "hoverkite";
+const HOMIE_DEVICE_NAME: &str = "Hoverkite";
+
+pub struct Homie<'a> {
+    homie: Option<HomieDevice>,
+    handle: &'a Handle,
+}
+
+impl<'a> Homie<'a> {
+    pub fn connect_and_start(
+        handle: &'a Handle,
+        config: Option<MqttConfig>,
+    ) -> Result<Homie<'a>, Report> {
+        let homie = if let Some(config) = config {
+            Some(handle.block_on(make_homie_device(config))?)
+        } else {
+            None
+        };
+        Ok(Self { homie, handle })
+    }
+
+    pub fn send_target(&self, side: Side, target: i64) {
+        self.send_property(node_id(side), "target", target);
+    }
```

Comment:
can we somehow disable github annotations for code coverage, so I don't have to disable them each time I review a PR? (I still want to see clippy annotations, if there are any)

Reply:
I'll look into it.